### DISCUSSION TITLE
Seal skills - Post battle implementation

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#333] [General]
I'm taking a stab at making the seal skills work perpetually instead of being based on a turn count. 

Given that, how to do I stop the debuff from resetting once the next turn starts? I don't see any calls to ``ClearUnitStatDebuff`` that could be causing it 🤔 

![image](https://github.com/user-attachments/assets/459a6bf2-2319-4c5e-bdab-ebc40b083bf2)

**Comment by MokhaLeee** [comment]
It use `TickUnitStatDebuff`

**Reply by JesterWizard** [reply]
Ah awesome, so I just needed to set the ``tick type`` to ``STATUS_DEBUFF_NO_TICK`` in ``StatDebuffInfo.c``.

(Just posting the solution for anyone else who may come across this).
